src/Microsoft.TemplateEngine.Edge/LocalizableStrings.resx

```diff
@@ -198,6 +198,9 @@
   <data name="GlobalSettingsTemplatePackageProvider_InstallResult_Error_PackageAlreadyInstalled" xml:space="preserve">
     <value>{0} is already installed.</value>
   </data>
+  <data name="GlobalSettingsTemplatePackageProvider_InstallResult_Error_DuplicatedIdentity" xml:space="preserve">
+    <value>The package with the same identity is already installed: {0}. Please consider uninstalling the conflicting template.</value>
```

Comment:
We need to tell the user what other package includes the duplicate template identity to help them decide which one they should uninstall. Maybe something like:

> The template {templateId} is already installed from template package {packageId}. Either uninstall {packageId} with {sample command}, or forcefully install this package by adding the `--force` flag, like {sample command}.

Reply:
Agree, however this should happen on dotnet/sdk side. Templating doesn't know the action to uninstall the template in SDK.

Once SDK gets this error code, it should determine duplicate identity, it's package and create command for uninstallation.

Reply:
Do I understand right that the message suggested by @baronfel has to be added on the sdk side?